tests/unit/test_testing_farm.py

```diff
@@ -198,7 +198,8 @@ def test_testing_farm_response(
     "chroot,compose,arch",
     [
         ("fedora-33-x86_64", "Fedora-33", "x86_64"),
-        ("centos-stream-x86_64", "Centos-Stream", "x86_64"),
+        ("epel-8-x86_64", "CentOS-8", "x86_64"),
+        ("centos-stream-x86_64", "CentOS-Stream", "x86_64"),
```

Comment:
I believe this should map to `CentOS-Stream-8` now:

https://api.dev.testing-farm.io/v0.1/composes

Reply:
thanks, fixed